### ISSUE TITLE
Icons with missing tooltips

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/tc-create-app#475]
![image.png](https://images.zenhubusercontent.com/56e1a9316caf56c9d38d8df5/22e9e476-d222-42dd-bdf8-efcfae541a20)

![image.png](https://images.zenhubusercontent.com/56e1a9316caf56c9d38d8df5/2da048db-c4ed-4f78-9fbb-dcff8ff8110f)

**Comment by birchamp** [comment]
@mandolyte I believe you're looking into a regression on tooltips you added.  Were these particular icons above ones that already had tooltips?

**Comment by mandolyte** [comment]
No, I don't think these were there when I was doing the original tooltip work. So these can be added.
